Dockerfile

```diff
@@ -1,9 +1,30 @@
-FROM node:14-buster
-LABEL maintainer="HMPPS Digital Studio <info@digital.justice.gov.uk>"
+FROM node:14-buster as builder
 
 ARG BUILD_NUMBER
 ARG GIT_REF
 
+RUN apt-get update && \
+    apt-get upgrade -y
+
+WORKDIR /app
+
+COPY package.json .
+COPY package-lock.json .
+ADD ./src/ src/
+ADD ./scripts/ scripts/
+ADD ./static/ static/
+ADD ./html-template/ html-template/
+COPY *.js ./
```

Comment:
could this be managed by a .dockerignore file?

Reply:
also, maybe could just copy everything - as the build stage gets binned at the end anyway.  I notice there is a copy everything step in the final stage also.  Maybe fine tune the `.dockerignore`  file also just to make sure we are not copying rubbish.

Reply:
agreed, will take a look in a future PR